test_work/run_mig.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+VER=1.15.pre
+DBCFG=../database.yml.local
+SEEDSQL=../mdummy_test.sql
```

Comment:
Configure the script here to load a different base version. Configure location of db.yml prototype.  You may have to adjust the load command for your user/dbname (mdummy_test/development, user decko_user).

In any case, this is so much to run as to give an almost working example for developers to use.